DockerFileTests

```diff
@@ -1,17 +1,8 @@
-FROM python:latest
-ARG GCLOUD_KEY
-ARG PROJECT_NAME
-ARG CLUSTER_NAME
-ARG CLUSTER_ZONE
-
-COPY . .
-
-ADD  . /go-spacemesh
+FROM python:latest AS tests_base
 
 WORKDIR /go-spacemesh/tests
```

Comment:
If you omit the commands: "COPY . ." and "ADD . /go-spacemesh" How can you set WORKDIR to /go-spacemesh/tests in this layer?

Reply:
According to [the docs](https://docs.docker.com/engine/reference/builder/#workdir):

> If the WORKDIR doesn’t exist, it will be created [...]

Reply:
OK. but then you copy tests under /go-spacemesh/tests which result in /go-spacemesh/tests/tests

Reply:
Again, from [the docs](https://docs.docker.com/engine/reference/builder/#copy):

> If `<dest>` does not end with a trailing slash, it will be considered a regular file and the contents of `<src>` will be written at `<dest>`.

(There's more context in the docs that you can read, like what happens if there is a trailing slash on `<dest>`.)

I wouldn't be 100% sure about this from the docs, but if I didn't understand it correctly and the structure turned out wrong - running the tests wouldn't work (it does).